org.eclipse.m2e.archetype.common/META-INF/p2.inf

```diff
@@ -0,0 +1,14 @@
+requires.0.namespace = osgi.bundle
```

Comment:
Can't we add generic provided capabilities to the fragments and require that in the manifest instead of using p2-instructions?